annotation/src/main/java/org/cbioportal/genome_nexus/annotation/GenomeNexusAnnotation.java

```diff
@@ -102,4 +114,85 @@ private ApiInfo apiInfo() {
             .version("2.0")
             .build();
     }
+
+    /**
+     * Export a resource embedded into a Jar file to the local file path.
+     *
+     * @param resourceName ie.: "/SmartLibrary.dll"
+     * @return The path to the exported resource
+     * @throws Exception
+     */
+    static public String ExportResource(String resourceName) throws Exception {
```

Comment:
It is against Java coding conventions to have a method name starting with an uppercase letter :) Can we rename it to `exportResource`?

Reply:
@onursumer ah yes, this function was a copy+paste from SO. Guess this was some C++/C# dev: https://www.reddit.com/r/java/comments/3cu10n/do_any_of_you_start_static_methods_with_a_capital/